src/headers/mq_op.h

```diff
@@ -34,12 +34,27 @@
 
 #define MAX_OPENQ_ATTEMPS 15
 
+#define SEND_MSG_STR "Sending message from %s: '%s'"
```

Comment:
Change the log format to `Sending %s event: '%s'`

Examples:
`Sending FIM event`
`Sending AWS integration event`